### PR TITLE
First part of the update for the Russian documentation

### PR DESCRIPTION
**Pull request by theartkod**
Ever since version 1.*, Russian version of the documentation has been outdated. This is the first part of updates and follow-up support.

Complete re-translation and update:
- /homepage
- /guides

### Thanks to

- @DreaMinder 
- @icehaunter 
- @sp1r1don 
- @merzavko 
- @Domovikx 
- @cogor 
- @Alex-Sokolov 
- @MaximStone

PS: _Take note of the new maintainers, translation effort was moved to [another group](https://github.com/theartkod/docs) as the original group is no longer active._

### CODE REVIEW

File: en/homepage/modes_statically_generated.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Statically Generated
 content_title: Statically Generated
 ---              
 Static Site Generation is a very hot topic right now (aka JAMStack). Instead of switching to another framework and
-spending time to get used to it, why not kill two birds with one stone?
+spending time to get used to it, why not killing two birds with one stone?
```
Comment (aldarund) [inline en/homepage/modes_statically_generated.md:6]: why? Dont think its supposed to be in this PR or right change at all.

Comment (theartkod) [reply en/homepage/modes_statically_generated.md]: fixed. some merge errors

**Comment by aldarund** [comment]
I`l check this PR, its quite big :)

**Comment by debs-obrien** [comment]
Thanks for your contributions. We are now closing this repo as docs have now moved to nuxtjs.org using content module. Please continue your contributions on that site especially focusing on the guides folder which is the new docs. Thanks

https://github.com/nuxt/nuxtjs.org/blob/master/README.md

**Pull request closed without merging**